suds/resolver.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, wsdl, ps='.'):
         Resolver.__init__(self, wsdl.schema)
         self.wsdl = wsdl
         self.altp = re.compile('({)(.+)(})(.+)')
-        self.splitp = re.compile('({.+})*[^\%s]+' % ps[0])
+        self.splitp = re.compile('({.+})*[^\\%s]+' % ps[0])
```

Comment:
I actually think the `\` can just be removed - it seems like the intent was to prevent `%s` from being treated as a string format (ie https://pyformat.info/#simple)

Reply:
```
$ python
Python 3.7.6 (default, Jan 30 2020, 09:44:41) 
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> '\\%s' % 'test'
'\\test'
>>> '\%s' % 'test'
'\\test'
>>> 
```

Perhaps this is not what's expected, but that should probably be fixed in a separate change, no?

Reply:
Interesting, thanks!